src/main/kotlin/sc/iview/SciView.kt

```diff
@@ -1796,6 +1797,17 @@ fun deleteNode(node: Node?, activePublish: Boolean = true) {
         return availableLUTs.map {entry -> entry.key}
     }
 
+    /**
+     * Return ProjectDirectories for sciview.
+     *
+     * Use this to find the location of cache, etc.
+     *
+     * @return a [ProjectDirectories] for sciview
+     */
+    fun getProjectDirectories(): ProjectDirectories {
```

Comment:
We might want to gather all sciview options into an options/config class, accessible via `sciview.getOptions()` or some such. This class could have the `ProjectDirectories` and also have any other state for managing configuration. What do you think? Is there significant other configuration to be managed?

Reply:
Mouse and key config is blurred between scenery and sciview, which is part of the issue with folks current input config concerns. I'm looking at the most surgical way of ensuring consistent input controls, but there are a lot of layers here...